pkgs/tools/networking/nordvpn/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, lib, fetchurl, openvpn, libxml2, autoPatchelfHook, dpkg }:
+
+let
+  version = "3.10.0-1";
+in stdenv.mkDerivation {
+  pname = "nordvpn";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://repo.nordvpn.com/deb/nordvpn/debian/pool/main/nordvpn_${version}_amd64.deb";
+    sha256 = "BNAInjJlQsYpxfUKI13oK/P6n6gpBlvgSQoJAuZ3C2M=";
+  };
+
+  nativeBuildInputs = [ libxml2 autoPatchelfHook dpkg ];
+
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" "distPhase" ];
+
+  unpackPhase = "dpkg -x $src unpacked";
+
+  installPhase = ''
+    mkdir -p $out/
+    sed -i 's;ExecStart=.*;;g' unpacked/usr/lib/systemd/system/nordvpnd.service
```

Comment:
Good idea. We can just replace the FHS path with the derivation path and drop the custom `ExecStart` in the `.service`. Either way.